src/components/tools/geometry-tool/axis-settings-dialog.sass

```diff
@@ -1,21 +1,23 @@
-.axis-settings-dialog
-  &.bp3-dialog
-    width: 600px
 
-  .nc-attribute-name-prompt
-    .axis-prompt
-      font-weight: bold
-      &.x-axis
-        margin-left: 120px
+.axis-settings-container
+  margin-top: 15px
+  margin-bottom: 15px
 
-      &.y-axis
-        margin-left: 150px
+  .axis-title
+    font-size: 12pt
+    font-weight: bold
+    margin-top: 10px
+    margin-bottom: 10px
 
-    .annotation-prompt
-      margin-right: 5px
+  .axis-options
+    display: flex
 
-    .min-prompt
-      margin-right: 15px
+    .axis-option
+      display: flex
+      flex-direction: column
 
-    .max-prompt
-      margin-right: 13px
+.label-padding
+  padding: .5em
+
+.input-margin
+  margin: 0 .5em
```

Comment:
This wasn't mentioned in the original story, but since you're having to update _all_ of the CSS in some of these files, it would be better to convert them to `.scss` files, which generally means adding semicolons and changing the file extension. We've been gradually migrating over time and this seems like a reasonable time to migrate these files.